tools/wptrunner/wptrunner/executors/base.py

```diff
@@ -80,9 +80,10 @@ def __call__(self, test, result, extra=None):
 testharness_result_converter = TestharnessResultConverter()
 
 
-def hash_screenshot(data):
+def hash_screenshots(screenshots):
```

Comment:
So our 'hash' is now a list of hashes; that's probably ok, but are we gaining anything from doing a hexdigest of each screenshot now or should we just return a list of base64 encodings?

Also nit, the docstring is now wrong.